changelogs/unreleased/ghs-117-possible-segfault-in-multiSelectOrderBy.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
```

Comment:
Looks more like bugfix/sql.

Reply:
Fixed.

Reply:
Come to think of it, the changelog looks useless, see https://github.com/tarantool/tarantool/pull/8454/commits/08b7bc9daffa6547af8e383df61c58168c182912#r1139924387. Let's just drop it?

Reply:
Done.